.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Want improvements or new features in SCT?
+
+---
+
+Hi, thanks for opening a feature request, please take some time to consider the applicable guidelines:
+https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst#reporting-a-bug-or-a-requesting-a-feature
```

Comment:
the URL didn't work, this did the job: 
https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst#reporting-a-bug-or-requesting-a-feature